ui/src/app/views/dataset/component.tsx

```diff
@@ -133,6 +266,18 @@ export const DataSet = withCurrentWorkspace()(class extends React.Component<
       onClose={() => {this.setState({ creatingConceptSet: false}); }}
       conceptDomainList={conceptDomainList}
       existingConceptSets={conceptSetList}/>}
+      {console.log(resource)}
+      {this.state.confirmDeleting &&
+      // This throws a type error in the logs, figure out how to fix that
+      //  Functions as it's supposed to
```

Comment:
you might be able to use the cast `____ as unknown as _____` pattern? Is it from fetch vs generated?

Reply:
Typescript compilation errors.

Reply:
Fixed